Sources/SwiftProtobuf/TextFormatEncoder.swift

```diff
@@ -145,13 +172,19 @@ internal struct TextFormatEncoder {
                 append(staticText: "inf")
             }
         } else {
-            // TODO: Be smarter here about choosing significant digits
-            // See: protoc source has C++ code for this with interesting ideas
             if let v = Int64(safely: value) {
                 appendInt(value: v)
```

Comment:
Since int64 in JSON use strings, I'm still a little worried that large values here could have trouble depending on what language the receiver is written in.  But I guess we deal with that when the time comes.